eframe/src/native/run.rs

```diff
@@ -230,13 +234,16 @@ pub fn run_wgpu(
         painter
     };
 
+    let render_state = painter.get_render_state().expect("Uninitialized");
```

Comment:
This always panics on Android, so this breaks the support added in https://github.com/emilk/egui/pull/1634.

Reply:
Well that's not cool! I don't have an Android to test on unfortunately.

Still, I did try to build it for Android real quick. A quick look over and, it appears that there is multiple things breaking the ability to build `eframe` for android right now, that isn't related to this PR. It looks like #1634 might not have actually gotten `eframe` building for Android, just `egui_wgpu` and `egui_winit`.

Some things I noticed need to be fixed for `eframe` will build for Android:

- The inclusion of the `arboard` crate for clipboard support needs to be made optional, because it doesn't build for android. 
- Same fix for the `glutin` dependency
- It appears that the `winit::event::Event::Paused` event doesn't exist, but it's used in the android event loop code.

For what I can tell, it looks like the `eframe` crate hasn't ever actually worked for Android yet, but it might not be a big deal to make work. I'm not sure if I'll be able to get the time to try to get it working yet, though.

Reply:
Cool! We shouldn't actually need glutin anyway if we are using the WGPU backend. I'm not sure why text wouldn't be working.

Reply:
Nice to hear some progress on this! Feel free to open a PR to add `cargo check --target aarch64-linux-android` to CI, and any other needed changes